spec/graphql/query/executor_spec.rb

```diff
@@ -132,6 +132,22 @@
       end
     end
 
+    describe 'if nil is given for a non-null field' do
+      let(:query_string) {%| query noMilk { cantBeNullButIs }|}
+      it 'turns into error messages' do
+        expected = {
+          "data" => { "cantBeNullButIs" => nil },
```

Comment:
Huh, 

> ...then the result of evaluating the entire field set that contains this field is now null

So, all of its siblings should be be ignored, it would be `"data" => nil` ?